README.md

```diff
@@ -1,6 +1,9 @@
-﻿#Microsoft Azure Active Directory Authentication Library (ADAL) for Android
+﻿# Microsoft Azure Active Directory Authentication Library (ADAL) for Android
 ===========
 
+| [Code Samples](https://github.com/azure-samples?utf8=✓&q=active-directory-android) | [Reference Docs](http://javadoc.io/doc/com.microsoft.aad/adal/1.11.0) | [Developer Guide](https://aka.ms/aaddev)
```

Comment:
adal will stay on 1.x I believe, can we point it to always pull in the latest 1.x version?